identity.tf

```diff
@@ -14,3 +14,10 @@ resource "azurerm_role_assignment" "identity" {
 
   role_definition_name = "Key Vault Secrets User"
 }
+
+resource "azurerm_role_assignment" "certificates_access" {
+  principal_id = azurerm_user_assigned_identity.identity.principal_id
+  scope        = data.azurerm_key_vault.main.id
+
+  role_definition_name = "Key Vault Certificates Officer"
```

Comment:
`Key Vault Secrets User`  on line 15 should do this?

Reply:
feel free to run from a branch and see if it fixes it but I don't think it will (and double check `Key Vault Secrets User` is in place currently)

Reply:
Yes  - Key Vault Secrets User is in place, yesterday I have manually added `Key Vault Certificates Officer` for Demo and that has fixed it for them, let me run from a branch for different environment and confirm with requester